spec/spec_helper_acceptance.rb

```diff
@@ -2,6 +2,27 @@
 require 'mustache'
 require 'open3'
 
+unless ENV['DEVELOPMENT']
+  require 'beaker-rspec'
+  require 'beaker/puppet_install_helper'
+  unless ENV['BEAKER_provision'] == 'no'
+    install_pe
+    on(default, puppet('module install puppetlabs-aws'))
+
+    # install the AWS SKD & retries gems on the agent
+    on(default, puppet('resource package aws-sdk-core ensure=installed provider=puppet_gem'))
+    on(default, puppet('resource package retries ensure=installed provider=puppet_gem'))
+
+    # install aws creds on SUT
+    home = ENV['HOME']
+    file = File.open("#{home}/.aws/credentials")
+    agent_home = on(default, 'printenv HOME').stdout.chomp
+    on(default, "mkdir #{agent_home}/.aws")
+    create_remote_file(default, "#{agent_home}/.aws/credentials", file.read)
+    # TODO enable promotion of artifact to modules forge
```

Comment:
Updated to use `puppet_module_install`.

Note that I also removed the forge api setting in the config as otherwise this doesn't work. The relevant logic is doing as intended, but not as I expected:

https://github.com/puppetlabs/beaker/blob/9631b55a2674032bef253e3a527715e2cfcad9b4/lib/beaker/dsl/install_utils/module_utils.rb#L26